src/content/en/updates/2019/02/trusted-types.md

```diff
@@ -0,0 +1,221 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Trusted Types is a new experimental API available in Chrome that helps prevent DOM-Based Cross-Site Scripting in your applications.
+
+{# wf_updated_on: 2019-02-07 #}
+{# wf_published_on: 2019-02-06 #}
+{# wf_tags: news,security,trusted-types,origintrials #}
+{# wf_featured_image: /web/updates/images/generic/security.png #}
+{# wf_featured_snippet: Trusted Types is a new experimental API available in Chrome that helps prevent DOM-Based Cross-Site Scripting in your applications. #}
+{# wf_blink_components: N/A #}
+
+# Trusted Types help prevent Cross-Site Scripting {: .page-title }
+
+#### TL;DR {: #tldr .hide-from-toc }
+We've created a new experimental API that aims tp prevent DOM-Based Cross
+Site Scripting in modern web applications.
+
+{% include "web/_shared/contributors/koto.html" %}
+
+<aside class="caution"> We’re currently working on the specification and
+implementation details for this API. We’ll keep this post updated as Trusted
+Types mature. </aside>
+
+## Cross-Site Scripting {: #xss }
+
+[Cross-Site Scripting](https://en.wikipedia.org/wiki/Cross-site_scripting) (XSS)
+is the most prevalent vulnerability affecting web applications. We see this
+reflected both in
+[our own data](https://security.googleblog.com/2016/09/reshaping-web-defenses-with-strict.html),
+and
+[throughout the industry](https://www.hackerone.com/sites/default/files/2017-06/The%20Hacker-Powered%20Security%20Report.pdf){: .external}.
+Practice shows that maintaining an XSS-free application is still a difficult
+challenge, especially if the application is complex. While solutions for
+preventing server-side XSS are well known,
+[DOM-based Cross-Site Scripting](https://www.owasp.org/index.php/DOM_Based_XSS){: .external}
+(DOM XSS) becomes a *growing* problem. For example, in
+[Google Vulnerability Reward Program](https://g.co/vrp) DOM XSS is already the
```

Comment:
either "Google's Vulnerability Reward Program" or "the Google Vulnerability Reward Program"